examples/main.tf

```diff
@@ -0,0 +1,137 @@
+
+provider "aws" {
+  region = var.aws_region
+}
+
+# VPC Module
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "~> 5.0.0"
+
+  name                 = local.vpc_name
+  cidr                 = var.vpc_cidr
+  private_subnets      = var.private_subnets
+  azs                  = var.availability_zones
+  enable_dns_hostnames = true
+  enable_dns_support   = true
+
+  tags = {
+    Name        = "example-vpc"
+    Environment = "dev"
+  }
+}
+
+module "ecs_platform" {
+  source                  = "terraform.registry.launch.nttdata.com/module_collection/ecs_appmesh_platform/aws"
```

Comment:
with source, you must use version too. applicable to all modules that are pulled from tf registry. not applicable for modules pulled locally or directly from github